README.md

```diff
@@ -67,6 +67,17 @@ kagglehub.model_download('google/bert/tensorFlow2/answer-equivalence-bem', path=
 kagglehub.model_download('google/bert/tensorFlow2/answer-equivalence-bem', force_download=True)
 ```
 
+### Upload Model
+To upload a model to Kaggle, use the `model_upload` function. This function allows you to upload model files to a specified handle on Kaggle.
+
+```python
+import kagglehub
+
+kagglehub.model_upload(handle='your/model/handle',
```

Comment:
This would fail because `license_name` is mandatory. However, once #62 is merged, it will be optional.

Can you give examples with and without version_notes given this is optional. Given that #62 will be merged soon. Let's do the same for license_name.

Also, make sure to use a better handle in your example.

For example, use:

```python
import kagglehub

# For example, to upload a new variation to this model:
# - https://www.kaggle.com/models/google/bert/frameworks/tensorFlow2/variations/answer-equivalence-bem
# 
# You would use the following handle: `google/bert/tensorFlow2/answer-equivalence-bem`
handle = '<KAGGLE_USERNAME>/<MODEL>/<FRAMEWORK>/<VARIATION>'
local_model_dir = 'path/to/local/model/dir'

kagglehub.model_upload(handle, local_model_dir)

# You can also specify some version notes (optional)
kagglehub.model_upload(handle, local_model_dir, version_notes='improved accuracy')

# You can also specify a license (optional)
kagglehub.model_upload(handle, local_model_dir, license_name='Apache 2.0')
```

Reply:
oh, yeah i was thinking do it this way because the cl to make license optional is underway. updated now, thanks!